pygmt/tests/test_grd2cpt.py

```diff
@@ -0,0 +1,316 @@
+"""
+Tests for grd2cpt.
+"""
+import os
+
+import numpy as np
+import pytest
+from pygmt import Figure, grd2cpt
+from pygmt.datasets import load_earth_relief
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile
+from pygmt.helpers.testing import check_figures_equal
+
+TEST_DATA_DIR = os.path.join(os.path.dirname(__file__), "data")
+POINTS_DATA = os.path.join(TEST_DATA_DIR, "points.txt")
+
+
+@pytest.fixture(scope="module", name="points")
+def fixture_points():
+    """
+    Load the points data from the test file.
+    """
+    return np.loadtxt(POINTS_DATA)
+
+
+@pytest.fixture(scope="module", name="region")
+def fixture_region():
+    """
+    The data region.
+    """
+    return [10, 70, -5, 10]
+
+
+@pytest.fixture(scope="module", name="grid")
+def fixture_grid():
+    """
+    Load the grid data from the sample earth_relief file.
+    """
+    return load_earth_relief(registration="gridline")
```

Comment:
```suggestion
    return load_earth_relief()
```
No need to hardcode `registration="gridline"` here. The tests in [`test_makecpt.py`](https://github.com/GenericMappingTools/pygmt/blob/v0.2.1/pygmt/tests/test_makecpt.py#L34) uses "gridline" registration because it was the default in GMT 6.0 and we didn't want to update the baseline images for the tests using `@pytest.mark.mpl_image_compare` (see #451 for context). Default for GMT 6.1 (what we're using now) is "pixel" registration (see also longer discussion at #487 on what those two mean).